erpnext/controllers/queries.py

```diff
@@ -18,7 +18,7 @@ def employee_query(doctype, txt, searchfield, start, page_len, filters):
 	fields = get_fields("Employee", ["name", "employee_name"])
 
 	return frappe.db.sql("""select {fields} from `tabEmployee`
-		where status = 'Active'
+		where status in ('Active', 'Suspended')
```

Comment:
I don't think we should allow selecting Suspended employees.

Reply:
Allowed because they may be part of payroll process and several other features... like suspended employee can be the part of payroll and can have their salary slip. They may also have leave policy assignments, they may also appear in Leave allocation and lots more process....